idb/postgres/postgres_migrations.go

```diff
@@ -2204,3 +2205,32 @@ func ClearAccountDataMigration(db *IndexerDb, state *MigrationState) error {
 
 	return nil
 }
+
+// MakeDeletedNotNullMigration makes "deleted" columns NOT NULL in tables
+// account, account_asset, asset, app, account_app.
+func MakeDeletedNotNullMigration(db *IndexerDb, state *MigrationState) error {
+	db.accountingLock.Lock()
+	defer db.accountingLock.Unlock()
+
+	queries := []string{
+		"UPDATE account SET deleted = false WHERE deleted is NULL",
+		"ALTER TABLE account ALTER COLUMN deleted SET NOT NULL",
+		"UPDATE account_asset SET deleted = false WHERE deleted is NULL",
+		"ALTER TABLE account_asset ALTER COLUMN deleted SET NOT NULL",
+		"UPDATE asset SET deleted = false WHERE deleted is NULL",
+		"ALTER TABLE asset ALTER COLUMN deleted SET NOT NULL",
+		"UPDATE app SET deleted = false WHERE deleted is NULL",
+		"ALTER TABLE app ALTER COLUMN deleted SET NOT NULL",
+		"UPDATE account_app SET deleted = false WHERE deleted is NULL",
+		"ALTER TABLE account_app ALTER COLUMN deleted SET NOT NULL",
+	}
+
+	for _, query := range queries {
+		_, err := db.db.Exec(query)
+		if err != nil {
+			return fmt.Errorf("query: \"%s\" err: %w", query, err)
+		}
+	}
```

Comment:
use `sqlMigration()`

Reply:
hmm, but that wraps all the changes in one big transaction block, and it might be better not to, _if_ we make sure it's a fully safe sequence (see other comment)

Reply:
why does it matter?